graphitron-codegen-parent/graphitron-java-codegen/src/test/java/no/sikt/graphitron/queries/fetch/NodeDirectiveTest.java

```diff
@@ -49,7 +49,10 @@ static void tearDown() {
     void defaultCase() {
         assertGeneratedContentContains(
                 "root/default", Set.of(CUSTOMER_QUERY),
-                "row(_iv_nodeIdStrategy.createId(\"Customer\", _a_customer.fields(_a_customer.getPrimaryKey().getFieldsArray()))).mapping"
+                "private static SelectField<Customer> queryForQuery_customer(NodeIdStrategy _iv_nodeIdStrategy)",
+                "queryForQuery_customer(_iv_nodeIdStrategy)",
+                "row(_iv_nodeIdStrategy.createId(\"Customer\", _a_customer.fields(_a_customer.getPrimaryKey().getFieldsArray()))).mapping",
+                "queryForQuery_customer(NodeIdStrategy _iv_nodeIdStrategy)"
```

Comment:
```suggestion
                "private static SelectField<Customer> queryForQuery_customer(NodeIdStrategy _iv_nodeIdStrategy)",
                "queryForQuery_customer(_iv_nodeIdStrategy)",
                "row(_iv_nodeIdStrategy.createId(\"Customer\", _a_customer.fields(_a_customer.getPrimaryKey().getFieldsArray()))).mapping"
```

Den er vel inkludert i den første linja, hvis jeg ikke ser helt feil nå

Reply:
du ser rett! Jeg fjerner heller den første jeg. Siden forventet output da er mer i tråd med rekkefølgen på den genererte koden